### DISCUSSION TITLE
skywalking ui traces list search by tags

### DISCUSSION THREAD

**Discussion by unihon** [apache/skywalking#12171] [Q&A]
question:
skywalking ui traces list search by tags sometime is no result

info:
![image](https://github.com/apache/skywalking/assets/42078581/ee250569-1688-4ce6-84b7-b16e9fc4dc47)
![image](https://github.com/apache/skywalking/assets/42078581/7b2f9ed9-40f6-44ff-9fe8-18b404fac092)

config:
```
searchableTracesTags: ${SW_SEARCHABLE_TAG_KEYS:http.method,http.status_code,rpc.status_code,db.type,db.instance,mq.queue,mq.topic,mq.broker,userName,http.status,status_code}
```

**Comment by wu-sheng** [comment]
Can't see what is your point of aaking this.
If you even don't invest time to locate the issue, there is nothing to answer.
